helm/chart/maesh/templates/mesh/mesh-daemonset.yaml

```diff
@@ -55,9 +55,10 @@ spec:
       {{- end }}
       containers:
         - name: maesh-mesh
-          image: {{ include "maesh.meshImage" . | quote }} 
-          imagePullPolicy: {{ .Values.mesh.image.pullPolicy | default "IfNotPresent" }}
+          image: {{ include "maesh.controllerImage" . | quote }}
```

Comment:
Maybe the `maesh.controller.image` property could be renamed to `maesh.image` as it is now used for the controller and proxy? The `maesh.controller.image` should be deprecated and still usable though.

WDYT?

Reply:
After a discussion with @SantoDE, we think that it's better to bump a major version of the Helm Chart (`v2.0.0`) and replace the `maesh.controller.image` and `maesh.mesh.image` properties by the `maesh.image` property. Even if it's possible to deprecate them the meaning of the `maesh.mesh.image` is not the same because we're not using the Traefik image anymore.